static/js/components/LearnerSearch.js

```diff
@@ -10,13 +11,26 @@ import {
 } from 'searchkit';
 import Grid, { Cell } from 'react-mdl/lib/Grid';
 import Card from 'react-mdl/lib/Card/Card';
+import iso3166 from 'iso-3166-2';
 
 import LearnerResult from './search/LearnerResult';
 import CountryRefinementOption from './search/CountryRefinementOption';
 import FilterVisibilityToggle from './search/FilterVisibilityToggle';
 import HitsCount from './search/HitsCount';
 import type { Option } from '../flow/generalTypes';
 
+let makeTranslations: () => Object = () => {
```

Comment:
since 'translations' is a searchkit-specific term/concept, i think it would help to name this `makeSearchkitTranslations` and the associated variable `searchkitTranslations`